lib/helpers/helpers.js

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import _ from 'lodash';
+import { exec } from 'teen_process';
+import B from 'bluebird';
+
+const isWindows = (os.type() === 'Windows_NT');
+const _fs = {
+  lstat: B.promisify(fs.lstat),
+  readdir: B.promisify(fs.readdir)
+};
+
+async function getDirectories (rootPath) {
+  let files = await _fs.readdir(rootPath);
+  let dirs = [];
+  for (let file of files) {
+    let pathString = path.resolve(rootPath, file);
+    if ((await _fs.lstat(pathString)).isDirectory()) dirs.push(file);
+  }
+  // It is not a clean way to sort it, but in this case would work fine because we have numerics and alphanumeric
```

Comment:
looks like the comment needs to be broken up into more lines, seems longer than 80 chars